### DISCUSSION TITLE
Feature requests: Export ASN list, SANE integration, expiry date

### DISCUSSION THREAD

**Discussion by confuso** [jonaswinkler/paperless-ng#1016] [Feature Requests]
Hi,
thank you for working on paperless-ng, it helps me a lot organizing myself. I have three ideas for some features:
- export ASN list: Would it be possible to export a list of ASN + document title? I would like to print it and add it to the binder
- The consumption directory works fine. On the other hand, I experienced, that the scanner I bought for transforming my documents into the paperless version, didn't support export to a FTP directory. So I had to use SANE for scanning and then manually saving the file to the consumption directory. SANE can be nicely controlled via CLI, so wouldn't it be quite easy to implement it as a function in paperless-ng (similar to the drag+drop function)?
- I don't need to keep all documents forever. I can throw normal invoices after 1-2 years, or if a contract ends, I don't need it anymore. Adding an expiry date would be nice to keep the archive clean :)

Maybe you find some of these ideas useful. Thanks a lot
Best
confuso

**Comment by amenk** [comment]
It might be better for discussion to make one Feature request per subject :-)

1) Concerning "export ASN list" using the browser's print function after filtering the document list might come in handy. Unfortunately I did not find a way to filter the list by "ASN is not empty" and the print CSS might need some improvement. With the latter one I might be able to help, if we go this route.

**Reply by amenk** [reply]
Printing the ASN list should not be possible: https://github.com/jonaswinkler/paperless-ng/discussions/126#discussioncomment-770924
